pkg/tests/observability_install_test.go

```diff
@@ -112,6 +115,41 @@ func installMCO() {
 		utils.PrintAllMCOPodsStatus(testOptions)
 	}
 
+	By("Checking placementrule CR is created")
```

Comment:
Please add `TODO` to remove the placementrule patch once the placementrule is implemented in server foundation.

Reply:
done.